ixwebsocket/IXSocketServer.h

```diff
@@ -60,6 +60,11 @@ namespace ix
 
         void setTLSOptions(const SocketTLSOptions& socketTLSOptions);
 
+        int  getPort();
+        std::string getHost();
+        int getBacklog();
+        std::size_t getMaxConnections();
```

Comment:
std::size_t is a bit verbose but maybe it is always defined on Windows. I had some weird macro for it in other places in the code base.